components/automate-cli/pkg/verifyserver/server/api/v1/nfsmount_test.go

```diff
@@ -0,0 +1,353 @@
{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location is shared across given nodes","error_msg":"","resolution_msg":""}],"error":null},{"ip":"localhost","node_type":"automate","checks":[{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location found","error_msg":"","resolution_msg":""},{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location is shared across given nodes","error_msg":"","resolution_msg":""}],"error":null},{"ip":"localhost","node_type":"chef-infra-server","checks":[{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location found","error_msg":"","resolution_msg":""},{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location is shared across given nodes","error_msg":"","resolution_msg":""}],"error":null},{"ip":"localhost","node_type":"chef-infra-server","checks":[{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location found","error_msg":"","resolution_msg":""},{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location is shared across given nodes","error_msg":"","resolution_msg":""}],"error":null},{"ip":"localhost","node_type":"postgresql","checks":[{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location found","error_msg":"","resolution_msg":""},{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location is shared across given nodes","error_msg":"","resolution_msg":""}],"error":null},{"ip":"localhost","node_type":"postgresql","checks":[{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location found","error_msg":"","resolution_msg":""},{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location is shared across given nodes","error_msg":"","resolution_msg":""}],"error":null},{"ip":"localhost","node_type":"postgresql","checks":[{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location found","error_msg":"","resolution_msg":""},{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location is shared across given nodes","error_msg":"","resolution_msg":""}],"error":null},{"ip":"localhost","node_type":"opensearch","checks":[{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location found","error_msg":"","resolution_msg":""},{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location is shared across given nodes","error_msg":"","resolution_msg":""}],"error":null},{"ip":"localhost","node_type":"opensearch","checks":[{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location found","error_msg":"","resolution_msg":""},{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location is shared across given nodes","error_msg":"","resolution_msg":""}],"error":null},{"ip":"localhost","node_type":"opensearch","checks":[{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location found","error_msg":"","resolution_msg":""},{"title":"NFS Mount","passed":true,"success_msg":"NFS mount location is shared across given nodes","error_msg":"","resolution_msg":""}],"error":null}]}`,
```

Comment:
Please format the expected string. Like actual string. It will be more readable.